apps/projects/app/components/Content/FilterTile.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Badge } from '@aragon/ui'
+
+import IconX from '../shared/assets/components/IconX'
+
+export default class FilterTile extends PureComponent {
+  static propTypes = {
+    text: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+    disableFilter: PropTypes.func.isRequired,
+  }
+  render() {
+    return (
+      <div style={{ marginLeft: '4px' }}>
+        <Badge.Identity>
+          <div style={{
+            paddingLeft: '10px',
+            paddingTop: '4px',
+            paddingBottom: '4px',
+            display: 'flex',
+            flexDirection: 'row'
+          }}>
+            {this.props.text}
+            <div
+              style={{
+                marginLeft: '10px',
+                paddingRight: '4px',
+                cursor: 'pointer'
+              }}
+              role="button" aria-pressed="false"
+              onClick={this.props.disableFilter}
```

Comment:
I usually get some linter settings here that give more instructions. I'm not sure if i missed some cross-site script blocking tags